Mathlib/Algebra/Divisibility/Basic.lean

```diff
@@ -0,0 +1,193 @@
+/-
+Copyright (c) 2014 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Amelia Livingston, Yury Kudryashov,
+Neil Strickland, Aaron Anderson
+-/
```

Comment:
```suggestion
Ported by: Matej Penciak
-/
```
We haven't been particularly thorough about this, but I'm encouraging people to add this now.